synapse/lib/storm.py

```diff
@@ -1319,3 +1320,12 @@ def _stormOperFromTags(self, query, oper):
                 [query.add(n) for n in nodes]
 
         return
+
+    def _stormOperGuid(self, query, oper):
+        args = oper[1].get('args')
+        core = self.getStormCore()
+
+        for iden in args:
+            node = core.getTufoByIden(iden)
```

Comment:
lets use getTufosByIdens here to minimize round trips and allow storage layer xact optimization